regen-network/regen-web
# feat: prefinance project card

## Description

Closes: #2276



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



- See featured projects section on https://deploy-preview-2288--regen-marketplace.netlify.app/
- https://deploy-preview-2288--regen-marketplace.netlify.app/profiles/regen18xc64ltgtu7uthkpj4fug2dnm8z5m0narvy0ww/projects

Prefinance projects on the projects/ list page will be handled separately as part of #2277

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
